operations/configuration_management/iSCSI_SBPS_Node_Customization.md

```diff
@@ -0,0 +1,251 @@
+
+**E.g.:**
+```bash
+ncn-m001:/home/csm-config-management # cray cfs v3 sessions create --name iscsi-sbps-targets-config --configuration-name iscsi-sbps-targets-config --format json
+{
+  "ansible": {
+    "config": "cfs-default-ansible-cfg",
+    "limit": "",
+    "passthrough": "",
+    "verbosity": 0
+  },
+  "configuration": {
+    "limit": "",
+    "name": "iscsi-sbps-targets-config"
+  },
+  "debug_on_failure": false,
+  "logs": "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config",
+  "name": "iscsi-sbps-targets-config",
+  "status": {
+    "artifacts": [],
+    "session": {
+      "completion_time": null,
+      "ims_job": null,
+      "job": null,
+      "start_time": "2024-08-31T21:08:47",
+      "status": "pending",
+      "succeeded": "none"
+    }
+  },
+  "tags": {},
+  "target": {
+    "definition": "dynamic",
+    "groups": [],
+    "image_map": []
+  }
+}
+ncn-m001:/home/csm-config-management #
+```
+
+**Note: To delete any old/ stale session with similar name**
+
+When you want to recreate the new CFS session with same name with that of an existing session for any reason 
+it may not allow it with the error _"Error: Conflicting session name: A session with the name iscsi-sbps-targets-config already exists"._
+
+You can delete the old/ stale CFS session before creating the new CFS session again.
+
+**E.g.:**
+```bash
+ncn-m001:/home/csm-config-management # cray cfs v3 sessions delete iscsi-sbps-targets-config
+
+ncn-m001:/home/csm-config-management #
+```
+
+### **View CFS session / check status**
+See [View CFS Sessions](CFS_Sessions.md) for more info.
+
+**E.g.:**
+```bash
+ncn-m001:/home/csm-config-management # cray cfs v3 sessions describe  iscsi-sbps-targets-config
+debug_on_failure = false
+logs = "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config"
+name = "iscsi-sbps-targets-config"
+
+[ansible]
+config = "cfs-default-ansible-cfg"
+limit = ""
+passthrough = ""
+verbosity = 0
+
+[configuration]
+limit = ""
+name = "iscsi-sbps-targets-config"
+
+[status]
+artifacts = []
+
+[tags]
+
+[target]
+definition = "dynamic"
+groups = []
+image_map = []
+
+[status.session]
+completion_time = "2024-08-31T21:09:19"
+job = "cfs-b840455c-e919-4656-b64a-44d433f082dc"
+start_time = "2024-08-31T21:08:47"
+status = "complete"
+succeeded = "True"
+
+ncn-m001:/home/csm-config-management #
+```
+
+### **Track the Status of Ansible playbooks**
+Make sure that all the CFS play books have completed successfully.
+Look for the message "All playbooks completed successfully" in the cfs ansible container log.
+
+See [Troubleshooting](Track_the_Status_of_a_Session.md) for more info.
+
+**E.g.:**
+
+```bash
+ncn-m001:/home/csm-config-management #  kubectl get pods --no-headers -o custom-columns=":metadata.name" -n services -l cfsession=iscsi-sbps-targets-config
+cfs-b840455c-e919-4656-b64a-44d433f082dc-5d4jp
+ncn-m001:/home/csm-config-management #
+```
+
+```bash

```

Comment:
Same here, remove `bash` as it is causing the `for` in line 235 and all of line 241 to get highlighted in red.  This is example output, not a bash script.  You should probably remove `bash` from all the comments, since none of them are actually quoting bash scripts.